release.sh

```diff
@@ -1,23 +1,31 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-set -v -e -o pipefail
+set -eux -o pipefail
```

Comment:
We should standardize the writing to simplify reviews ;).
How about  `set -xeuo pipefail` and `set -euo pipefail`?